rest/blip_client_test.go

```diff
@@ -50,8 +51,12 @@ type BlipTesterClientOpts struct {
 
 	// sendReplacementRevs opts into the replacement rev behaviour in the event that we do not find the requested one.
 	sendReplacementRevs bool
+
+	revsLimit *int // defaults to 20
 }
 
+const defaultBelipTesterClientRevsLimit = 20
```

Comment:
```suggestion
// defaultBlipTesterClientRevsLimit is the number of revisions stored by the blip test client, old revisions will be discarded
const defaultBlipTesterClientRevsLimit = 20
```

not sure the comment is right but belip is definitely a typo